packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -360,12 +324,6 @@ class BondWithdrawalWatcher extends BaseWatcherWithEventHandlers {
       )
       const l2Bridge = this.getSiblingWatcherByChainId(destinationChainId)
         .bridge as L2Bridge
-      const hasPositiveBalance = await l2Bridge.hasPositiveBalance()
```

Comment:
@miguelmota curious if you think this is ok to remove since we have the `credit.lt(amount)` check right below which seemingly does the same thing.

Reply:
yes it's ok to remove